pvl/decoder.py

```diff
@@ -77,6 +137,13 @@ class PVLDecoder(object):
     decimal_chars = char_set('0123456789')
     hex_chars = char_set('0123456789ABCDEFabcdef')
 
+    def __init__(self):
```

Comment:
@godber I got rid of the keyword argument because I think that will preserve backwards compatibility for anyone that is making a subclass in order to decode